doc/source/serve/index.md

```diff
@@ -6,10 +6,8 @@
 
 # Serve: Scalable and Programmable Serving
 
-:::{tip}
-[Get in touch with us](https://docs.google.com/forms/d/1l8HT35jXMPtxVUtQPeGoe09VGp5jcvSv0TqPgyz6lGU) if you're using or considering using Ray Serve.
-
-Chat with Ray Serve users and developers on our [forum](https://discuss.ray.io/).
+:::{note}
+[Get in touch with us](https://docs.google.com/forms/d/1l8HT35jXMPtxVUtQPeGoe09VGp5jcvSv0TqPgyz6lGU) if you're using or considering using Ray Serve!
```

Comment:
Removed discuss link because there's already a discuss link at the top of the docs
<img width="839" alt="Screen Shot 2022-07-11 at 11 30 41 AM" src="https://user-images.githubusercontent.com/5459654/178333462-45d6b309-7250-44f8-a790-f496553e2e0c.png">